### ISSUE TITLE
Button styles

### ISSUE THREAD

**Issue by decentralandmiguel** [decentraland/profile#135]
I was wondering that the best way to solve and to adapt our buttons to the design proposal with semantic will be this decision.

Can we change all this set of buttons
![image.png](https://images.zenhubusercontent.com/64d100d037a41802ee2a6fe8/bea36925-a035-41e1-aa5a-29cbae3f4e86)

By the inverted button that we have in semantic? The white one.
![image.png](https://images.zenhubusercontent.com/64d100d037a41802ee2a6fe8/95199795-0261-4d49-afd6-a5e03a350c9e)

So in conclusion all the buttons have to be like the style of the button.

**Comment by LautaroPetaccio** [comment]
- Get unique name
- Activate world
- Social actions
should be primary, all the other actions should be inverted (secondary).
